HLTrigger/Timer/plugins/ThroughputService.cc

```diff
@@ -95,7 +96,7 @@ void ThroughputService::preGlobalBeginRun(edm::GlobalContext const& gc) {
     };
 
     // book MonitorElement's for this run
-    edm::Service<DQMStore>()->meBookerGetter(bookTransactionCallback);
+    edm::Service<dqm::legacy::DQMStore>()->meBookerGetter(bookTransactionCallback);
```

Comment:
Maybe this change is unimportant; again it follows what is done in the `FastTimerService`
https://github.com/cms-sw/cmssw/blob/c0d007c5a8e169da2cc6dfea20751bd7215fccb9/HLTrigger/Timer/plugins/FastTimerService.cc#L940

In `ThroughputService`, the following is used
https://github.com/cms-sw/cmssw/blob/c0d007c5a8e169da2cc6dfea20751bd7215fccb9/HLTrigger/Timer/plugins/ThroughputService.h#L32

Reply:
what difference does it make to use `dqm::legacy` instead of `dqm::reco` ?

Reply:
answering to myself: none, they are `typedef` one to the other.

Reply:
Thanks for the info. Since there is no difference, I will remove the extra `dqm::legacy::` from here.